BUILD.gn

```diff
@@ -437,6 +438,26 @@ static_library("electron_lib") {
     ]
   }
 
+  if (enable_tts) {
+    sources += [
```

Comment:
@deepak1556 what do you think, should be moved to `components/`?

Reply:
Yup `tts`, `printing`, `flash`, `spellchecker` and `widevine` are better as components.

Reply:
I investigated this a bit, and it looks like to me that `tts` could move to `components/` fairly easily, but printing depends on `BrowserProcess`, which still lives under `chromium_src` and has a global defined for the instance, which according to the component docs would break things. Feel free to correct me if I'm wrong.